database/sql/src/main/scala/cromwell/database/slick/tables/MetadataDataAccessComponent.scala

```diff
@@ -14,5 +15,17 @@ class MetadataDataAccessComponent(val driver: JdbcProfile) extends DataAccessCom
       customLabelEntries.schema ++
       metadataEntries.schema ++
       summaryStatusEntries.schema ++
-      workflowMetadataSummaryEntries.schema
+      workflowMetadataSummaryEntries.schema ++
+      summaryQueueEntries.schema
+
+  // Looks like here is the most appropriate place for this val since it doesn't fit neither in
+  // SummaryQueueEntryComponent nor in MetadataEntryComponent
+  val metadataEntriesToSummarizeQuery = {
+    Compiled(
+      (limit: ConstColumn[Long]) => (for {
+        (_, metadataEntry) <-
+          summaryQueueEntries join metadataEntries on (_.metadataJournalId === _.metadataEntryId)
```

Comment:
Is this join safe in edge cases like "a row exists in the summary queue but not in the metadata entry table"?

Reply:
I guess it should be fine, since locally I ran summarizer without exceptions against fresh clean `cromwell_test` db.